phpunit/class-register-block-type-from-metadata-test.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Test `register_block_type_from_metadata`.
+ *
+ * @package Gutenberg
+ */
+
+class Register_Block_Type_From_Metadata_Test extends WP_UnitTestCase {
+	/**
+	 * Tests that the function returns false when the `block.json` is not found
+	 * in the WordPress core.
+	 */
+	function test_metadata_not_found_in_wordpress_core() {
+		$result = register_block_type_from_metadata( 'unknown' );
+
+		$this->assertFalse( $result );
+	}
+
+	/**
+	 * Tests that the function returns false when the `block.json` is not found
+	 * in the current directory.
+	 */
+	function test_metadata_not_found_in_the_current_directory() {
+		$result = register_block_type_from_metadata( __DIR__ );
+
+		$this->assertFalse( $result );
+	}
```

Comment:
I still plan to add a test that covers block registration with the `block.json` fixture.

Reply:
Added.